Engine/Module/ModuleTime.cpp

```diff
@@ -116,11 +117,13 @@ void ModuleTime::Play()
 		SetTimeScale(1.f);
 		frame_start_time = game_time_clock->Read();
 		App->animations->PlayAnimations();
+		App->audio->ResetRTPCValues("FilterLPF");
```

Comment:
Better lets call this in a script, here we will be mixing game logic with engine